template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Nuget/.template.config/template.json

```diff
@@ -2,12 +2,20 @@
   "author": "Microsoft",
   "classifications": ["Config"],
   "name": "Nuget Config",
-  "tags": {
```

Comment:
Thinking about this more, I think we should leave `tags` in our templates for compat with the RTW release of the CLI (we'll be able to install these templates in that environment without updating the engine)